engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/PeriodicProcessService.scala

```diff
@@ -605,6 +609,8 @@ object PeriodicProcessService {
       runtimeStatusOpt: Option[StatusDetails]
   ) {
 
+    val runAtWithCreatedAt: (LocalDateTime, LocalDateTime) = (runAt, createdAt)
```

Comment:
AFAIS we create this `val` only for the purpose of sorting in this service. WDYT about creating Ordering (or even better, the cats' Order), well-named object and use `.sort(ourWellNamedOrder)` instead of `.sortBy` and this artificial `val`. WDYT?

Reply:
That is true, thanks - I've created companion object with implicit ordering (I've used Ordering since we already use it inside this scope) and now we can just use `sorted` and for `reverse` use reverted ordering rather than `reverse` on sorted list. WDYT of that approach?